packages/capnpc-ts/src/generators.ts

```diff
@@ -325,11 +326,18 @@ export function generateStructFieldMethods(
 
       }
 
+      const listClassIdentifier = ts.createIdentifier(listClass);
+
       adopt = true;
       disown = true;
       get = ts.createCall(
-        ts.createPropertyAccess(
-          THIS, '_getList'), __, [offsetLiteral, ts.createIdentifier(listClass)]);
+        ts.createPropertyAccess(THIS, '_getList'),
+        __,
+        [offsetLiteral, listClassIdentifier]);
+      init = ts.createCall(
```

Comment:
That's fine. I do have some git-fu for you though; add git@github.com:jdiaz5513/capnp-ts as a remote to your fork next time: https://help.github.com/articles/configuring-a-remote-for-a-fork/

Then after running a pull once you can easily cherry pick a commit:

```shell
git pull
git cherry-pick 32df29647c3f9dc181b9b62b9d4eaef308fd0292
```

Slaps that commit as-is on top of your HEAD, and lets you resolve if there's a merge conflict.